Docker/build/QuickStatements/README.md

```diff
@@ -4,14 +4,20 @@ Quickstatements2 as seen at  [https://github.com/magnusmanske/quickstatements](h
 
 ### Environment variables
 
-Variable                   | Default  | Description                                            
--------------------------- | -------- | -----------
-`WIKIBASE_SCHEME_AND_HOST` | NONE     | Host and port of wikibase instance as seen by QS container                                                  
-`WB_PUBLIC_HOST_AND_PORT`  | NONE     | Host and port of wikibase as seen by the user's browser
-`QS_PUBLIC_HOST_AND_PORT`  | NONE     | Host and port of QS as seen by the user's browser      
-`OAUTH_CONSUMER_KEY`       | NONE     | OAuth consumer key (obtained from wikibase)            
-`OAUTH_CONSUMER_SECRET`    | NONE     | OAuth consumer key (obtained from wikibase)            
-`PHP_TIMEZONE`             | UTC      | setting of php.ini date.timezone                       
+Variable                             | Default      | Description                                            
+-------------------------------------|--------------|------------
+`WIKIBASE_SCHEME_AND_HOST`           | NONE         | Host and port of wikibase instance as seen by QS
+`WB_PUBLIC_SCHEME_HOST_AND_PORT`     | NONE         | Host and port of wikibase as seen by the user's browser
+`QS_PUBLIC_SCHEME_HOST_AND_PORT`     | NONE         | Host and port of QS as seen by the user's browser      
+`OAUTH_CONSUMER_KEY`                 | NONE         | OAuth consumer key (obtained from wikibase)            
+`OAUTH_CONSUMER_SECRET`              | NONE         | OAuth consumer key (obtained from wikibase)            
```

Comment:
is this also consumer key (see line above)?

Reply:
Looks like some old blunder, good catch